devel/libgit2/Portfile

```diff
@@ -25,9 +25,9 @@ long_description    libgit2 is a portable, pure C implementation of the \
 
 homepage            https://libgit2.org/
 
-checksums           rmd160  3f9ced0e0dff170a8156e4aa8fcb0abce66c8f60 \
-                    sha256  5dae7cb32b6977cd95ed849d24f3692f0b7e9eb9b0ee9ffaa14baebb9cac76e1 \
-                    size    5304918
+checksums           rmd160  921ad95fe1941f638118a36cc9f1274555e38410 \
+                    sha256  14fa276cdc63edd900cae91a66df0b8c534434a69b9c5a0145b9e437eaf31486 \
+                    size    5419376
```

Comment:
This port uses the default "tarball" type GitHub download which is deprecated. If this project did not offer "release" type downloads, you should set `github.tarball_from archive` to use an "archive" type download instead. Since this project does offer "release" type downloads, those are preferred over both "tarball" and "archive" and you should set `github.tarball_from release`. Either way, the checksums will change, so `sudo port clean --all libgit2` to remove the previous "tarball" distfile you downloaded and then download the new one and recompute the checksums.